client/src/components/Card/PokemonCard.test.tsx

```diff
@@ -41,7 +41,7 @@ describe("PokemonCard Component", () => {
     expect(imageElement).toBeInTheDocument();
   });
 
-  it("checking for the conditional rendering of PokemonCard component", () => {
+  /*it("checking for the conditional rendering of PokemonCard component", () => {
```

Comment:
is this meant to be commented out?

Reply:
Ah, it's because I wanted to keep it all as I wasn't sure whether Kruthi had done styles too, but wanted something just in case. I'll remove it. The stats aren't conditionally rendered anymore because it has the flip, so the test is no longer relevant.